draft-hsts-tracking-prevention.md

```diff
@@ -0,0 +1,256 @@
+---
+###
+# Internet-Draft Markdown Template
+#
+# Rename this file from draft-todo-yourname-protocol.md to get started.
+# Draft name format is "draft-<yourname>-<workgroup>-<name>.md".
+#
+# For initial setup, you only need to edit the first block of fields.
+# Only "title" needs to be changed; delete "abbrev" if your title is short.
+# Any other content can be edited, but be careful not to introduce errors.
+# Some fields will be set automatically during setup if they are unchanged.
+#
+# Don't include "-00" or "-latest" in the filename.
+# Labels in the form draft-<yourname>-<workgroup>-<name>-latest are used by
+# the tools to refer to the current version; see "docname" for example.
+#
+# This template uses kramdown-rfc: https://github.com/cabo/kramdown-rfc
+# You can replace the entire file if you prefer a different format.
+# Change the file extension to match the format (.xml for XML, etc...)
+#
+###
+title: "HSTS Tracking Prevention"
+category: info
+
+docname: draft-hsts-tracking-prevention-latest
+submissiontype: IETF  # also: "independent", "editorial", "IAB", or "IRTF"
+number:
```

Comment:
What's this for?

Reply:
After a lot of digging I found this
https://www.rfc-editor.org/rfc/rfc7991.html#page-96

This is the "stream" that the document has been accepted into. After some reading I think this is better as "independent" for now.

Reply:
Eventually found https://www.rfc-editor.org/rfc/rfc7991.html#appendix-A.2 which describes this as the "stream" that the document belongs to.

It seems like independent is the best choice atm.